crowbar_framework/app/helpers/installer/upgrades_helper.rb

```diff
@@ -58,9 +58,7 @@ def upgrade_error_flash(message)
     end
 
     def upgrade_ha_repo_needed?
-      return true if Proposal.where(barclamp: "pacemaker").first
-
-      false
+      !!Proposal.where(barclamp: "pacemaker").first
```

Comment:
Ok, @rsalevsky @nicolasbock @aplanas , let me ask for feedback before I update the PR, otherwise we run through useless iterations.

not liked:
A: `!!Proposal.where(barclamp: "pacemaker").first`

same thing, but a bit longer:
B: `!Proposal.where(barclamp: "pacemaker").first.nil?`

boolean only test (even simpler):
C: `Proposal.where(barclamp: "pacemaker").size > 0`